src/main.rs

```diff
@@ -110,11 +131,29 @@ impl Op {
     fn print_root(&self, resolver: &Resolver) {
         match self {
             Op::Add(dep) | Op::Remove(dep) | Op::Update(_, dep) => {
-                let dir = resolver.dir(dep).map(|d| d.display().to_string()).unwrap_or_default();
+                let dir = resolver
+                    .dir(dep)
+                    .map(|d| d.display().to_string())
+                    .unwrap_or_default();
```

Comment:
I'm pretty sure `rustfmt` flattened this one to a single line. Your formatter probably follows some different set of rules?